apps/api/v2/src/modules/users/outputs/get-users.output.ts

```diff
@@ -0,0 +1,96 @@
+import { Type } from "class-transformer";
+import { Expose } from "class-transformer";
+import { IsBoolean, IsDateString, IsInt, IsString, ValidateNested, IsArray } from "class-validator";
+
+export class GetUserOutput {
+  @IsInt()
+  @Expose()
+  id!: number;
```

Comment:
when inputs/outputs class are defined outside of api-v2 nestjs cli is not able to auto generate the docs that's why we need to add @ApiProperty